Documentation/core-repos.md

```diff
@@ -1,41 +1,79 @@
-# .NET Core Repos
+# .NET Core repositories
```

Comment:
should this say just .NET since you're listing .NET Framework and other non-Core repos?

```suggestion
# .NET repositories
```

Reply:
Perhaps the file should be renamed too? Right now it's `core-repos.md`